pkgs/development/python-modules/cupy/default.nix

```diff
@@ -36,9 +38,20 @@ buildPythonPackage rec {
     wheel
   ];
 
+  checkInputs = [
+    pytest
```

Comment:
Can you try again if the tests work? Maybe also convert it to pytestCheckHook.

Reply:
Sure, but I highly doubt it, at least if the tests try to use the GPU.